src/mcp_atlassian/markdown_converter.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import tempfile
+from pathlib import Path
+
+from md2conf.converter import (
+    ConfluenceConverterOptions,
+    ConfluenceStorageFormatConverter,
+    elements_from_string,
+    markdown_to_html,
+)
+
+# Configure logging
+logger = logging.getLogger("mcp-atlassian.markdown_converter")
+
+
+def markdown_to_confluence_storage(markdown_content):
```

Comment:
Would you consider moving this to `preprocessing.py` since most of the preprocessing modules are located there?